sticker-utils/src/subcommands/print_labels.rs

```diff
@@ -0,0 +1,81 @@
+use std::fs::File;
+use std::hash::Hash;
+
+use clap::{App, Arg, ArgMatches};
+use failure::Fallible;
+use stdinout::OrExit;
+use sticker::encoder::deprel::{RelativePOSEncoder, RelativePositionEncoder};
+use sticker::encoder::layer::LayerEncoder;
+use sticker::encoder::SentenceEncoder;
+use sticker::serialization::CborRead;
+use sticker::wrapper::{Config, EncoderType, LabelerType, TomlRead};
+use sticker::Numberer;
+
+use crate::traits::StickerApp;
+
+const CONFIG: &str = "CONFIG";
+
+pub struct PrintLabelsApp {
+    config: String,
+}
+
+impl PrintLabelsApp {
+    fn print_labels_with_encoder<E>(&self, config: &Config) -> Fallible<()>
+    where
+        E: SentenceEncoder,
+        E::Encoding: Clone + Eq + Hash + ToString,
+        Numberer<E::Encoding>: CborRead,
+    {
+        let labels: Numberer<E::Encoding> = config.labeler.load_labels().or_exit(
+            format!("Cannot load label file '{}'", config.labeler.labels),
+            1,
+        );
+
+        for idx in 1..labels.len() {
```

Comment:
Doesn't this produce a mismatch between the number of labels in the logits vs the number of labels here?

Reply:
Could you explain this a bit further?

sticker does not currently only use 0 for padding, but it also creates the classification layer using `labels.len()`, so there first logit is used for padding. Since we never backprop padding, its probability should be near-0. Though, it can never be predicted, because `top_k_predictions` excludes this logit/probability.

Reply:
If I load a graph and use this label list to map into the logits, I'd be one-off due to the missing padding, right?

Reply:
That's why it prints the index as well (starting at 0). I would also be fine with printing an artificial PADDING label before looping. Would that be good enough?

Reply:
Sure, I prefer being explicit here.